src/jllvm/vm/Interpreter.hpp

```diff
@@ -252,12 +251,17 @@ class Interpreter : public OSRTarget
 
     /// Single entry for use in 'm_interpreterCCSymbols' as an implementation for ALL methods.
     std::uint64_t (*m_interpreterEntry)(const Method*, const std::uint64_t*){};
+    /// OSR Entry for frames with interpreter calling convention.
+    void* m_interpreterInterpreterCCOSREntry;
+    /// OSR Entry for frames with JIT calling convention returning a reference type.
+    void* m_interpreterJITCCOSREntryReferenceReturn;
+    /// OSR Entries for frames with JIT calling convention returning a base type.
+    std::array<void*, BaseType::MaxValue + 1> m_interpreterJITCCOSREntries{};
```

Comment:
Could you do something like:
```suggestion
    std::array<void*, BaseType::MaxValue - BaseType::MinValue + 1> m_interpreterJITCCOSREntries{};
```